lib/amazon_product_advertising_client.ex

```diff
@@ -7,7 +7,8 @@ defmodule AmazonProductAdvertisingClient do
   alias AmazonProductAdvertisingClient.Config
 
   @scheme "http"
-  @host   "webservices.amazon.com"
+  @host    Application.get_env(:amazon_product_advertising_client, :marketplace_host) ||
```

Comment:
[`Application.get_env/3`](https://hexdocs.pm/elixir/Application.html#get_env/3) takes an optional third parameter that is the default value in case the configuration doesn't exist. Please use that instead of `||`.